tests/Rules.Framework.IntegrationTests/InMemoryRulesDataSource.cs

```diff
@@ -26,8 +26,16 @@ public Task AddRuleAsync(Rule<TContentType, TConditionType> rule)
             TContentType contentType,
             DateTime dateBegin,
             DateTime dateEnd)
-            => Task.FromResult<IEnumerable<Rule<TContentType, TConditionType>>>(
-                this.rules.Where(r => object.Equals(r.ContentContainer.ContentType, contentType)).ToList());
+        {
+            var filteredByDate = this.rules.Where(r =>
```

Comment:
This code has been fixed in master.

Reply:
Ok, I added the Active now

Reply:
Keep all the && consistently placed (all either at the end of the line or the beggining of the next).

Reply:
I actually removed this filter by active in this place, I changed it to the RulesEngine